docs/usage/getting-started/private-packages.md

```diff
@@ -121,9 +121,11 @@ The following details the most common/popular manager artifacts updating and how
 
 ### composer
 
-Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse.
+Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse. For `github.com`, `hostType=git-tags` can be used to narrow down the use of the PAT for this use case only.
 Any `hostRules` with `hostType=packagist` are also included.
 
+If you are using a `github.com` repository make sure the hostRule contains a personnal access token, not an application token. The repository in `composer.json` should have the `vcs` type with a `https` url.
```

Comment:
```suggestion
If you are using a `github.com` repository make sure the hostRule contains a personal access token, not an application token.
The repository in `composer.json` should have the `vcs` type with a `https` url.
```

Is this referring to the case where people are running against non-github.com, e.g. they are running against self-hosted GitLab? It's unclear from this what users on github.com should do, especially if they are running as an application.

Reply:
If you run with the github application and your project have a dependency on a private github repository that is not hosted on packagist, there is ontly one way to make it work.

1) This pr must be merged
2) In composer.json the private repository must be referenced with the `vcs` type
3) Your renovate configuration must have a PAT for github on a hostRule with `hostType=git-tags` .

Do you want me to explicitely state this use case in a sub-section of the documentation ?

Reply:
I tried to improve the documentation like this : 

---

Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse.
Any `hostRules` with `hostType=packagist` are also included.
For dependencies on `github.com` without a packagist server, a hostRule with `hostType=git-tags` should be used with a personal access token (not an application token). A hostRule with `hostType=github` whould override the default renovate application token for everything else and cause unwanted side effects.

The repository in `composer.json` should have the `vcs` type with a `https` url. Ex:

```json
"repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/organization/private-repository",
    },
```